src/components/popover-container/popover-container.style.ts

```diff
@@ -6,6 +6,38 @@ import { baseTheme } from "../../style/themes";
 import IconButton from "../icon-button";
 import StyledIcon from "../icon/icon.style";
 
+function animationToRender({
+  animationState,
+  disableAnimation,
+}: {
+  animationState?: TransitionStatus;
+  disableAnimation?: boolean;
+}) {
+  if (disableAnimation) return "opacity: 1";
```

Comment:
```suggestion
  if (disableAnimation) return "opacity: 1;";
```

**suggestion**: I don't know if it matters here, but I'm pretty sure we've had issues in the past with forgetting the trailing `;` in lines of CSS in styled-components. Seems safer to include it to me.